### DISCUSSION TITLE
NutUI Taro 运行 h5 端 在 taro 中如何响应式单位转换 rem

### DISCUSSION THREAD

**Discussion by richard1015** [jdf2e/nutui#693] [Q&A]
https://docs.taro.zone/docs/config-detail#h5esnextmodules
配置需要额外的经由 Taro 预设的 postcss 编译的模块。
``` javascript
module.exports = {
  // ...
  h5: {
    // ...
    // 经过这一配置之后，代码中引入的处于 `node_modules/@nutui/nutui-taro/` 路径下的样式文件均会经过 postcss 的编译处理。
    esnextModules: ['nutui-taro']
  }
}
```

**Comment by bailongren** [comment]
module.exports = {
  // ...
  h5: {
    // ...
    // 经过这一配置之后，代码中引入的处于 `node_modules/@nutui/nutui-taro/` 路径下的样式文件均会经过 postcss 的编译处理,后续taro更新会也会支持正则表达，配置后如果出现页面字体统一变小，可结合设计稿尺寸或通过 'postcss-px-scale' 配置。
    esnextModules: ['nutui-taro']
    
  }
}

**Comment by Picamarichu** [comment]
请问大神，在微信小程序端希望把NutUI中的单位都转成rpx，也是这样配置嘛？

**Reply by Picamarichu** [reply]
> 请问大神，在微信小程序端希望把NutUI中的单位都转成rpx，也是这样配置嘛？

 mini: {
    publicPath: '/',
    staticDirectory: 'static',
    esnextModules: ['nutui-taro'],   这样配置了 并没有什么效果    ：（

**Reply by xyw1229** [reply]
后来解决了么？？

**Reply by xyw1229** [reply]
我解决了，发现config下的mini postcss配置里有个黑名单，去掉就好了
